### ISSUE TITLE
Drupal 9 compatibility

### ISSUE THREAD

**Issue by ccjjmartin** [fourkitchens/sous-drupal-project#8]
## Drupal 9 compatibility
- How ready is sous for Drupal 9

## Projects checked:
| Project | Status | Issue |
| --------- | --------- | -------- |
| allowed_formats | Not ready | https://www.drupal.org/project/allowed_formats/issues/3121483#comment-13532545 |
| drush/drush | Not ready | ??? had to remove to upgrade core |
| drupal/console | Not ready | ??? - had to remove to upgrade core |
| drupal/config_installer | Not ready | ??? - had to remove to upgrade core |
| drupal/config_direct_save | Not ready | ??? - had to remove to upgrade core |
| fourkitchens/sous-drupal-distro | Not ready | Need to create a PR for this |

**Comment by generalredneck** [comment]
@americkson @ccjjmartin, Can either of yall confirm that Sous is D9 ready? this seems to say it's not.

**Comment by ccjjmartin** [comment]
@generalredneck Adam created an excel spreadsheet that went way more into detail than this issue.  Rebeca worked on it as a rock a few quarters ago but last I checked there were something like 3 contrib modules that still needed patches made to be D9 ready.

**Comment by ccjjmartin** [comment]
@generalredneck Here it is: https://docs.google.com/spreadsheets/d/13SANpA7tz03_ZdNmto1FAwXowqbuGt8QbjR2Qp1JIB4/edit#gid=0

**Comment by generalredneck** [comment]
So part of the challenge is I didn't know my way around enough to distiguish the difference between this project and https://github.com/fourkitchens/sous-drupal-distro. I created https://github.com/fourkitchens/sous-drupal-distro/issues/18 as it is more pointed to the profile itself.

As for the dependencies, we can continue to maintain them over here because there is more... but it may require an additional issue to alter the composer.json file over on the profile side to incorporate all the patches and dev versions....

Updated the spreadsheet.
